core/jazz_acl/components/error-handler.js

```diff
@@ -18,30 +18,35 @@ module.exports = () => {
     const errorObj = {
         throwInputValidationError: (errorMessage) => (
             {
+                errorCode: 400,
```

Comment:
I  think `errorCode` here is not necessary here. API Gateway does a mapping of `errorType` value to HTTP status code.

Reply:
updated